src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1818,7 +1822,7 @@ export const handleUpdateLogic = [
         }),
         preventSubmitMessage: Joi.alternatives().conditional('logicType', {
           is: LogicType.PreventSubmit,
-          then: Joi.string().required(),
+          then: Joi.string(),
```

Comment:
can I check why this is not `required`?

Reply:
As above, it was to align with the schema, but I've tightened since we do require this on the frontend